src/api/repos/user-repo.ts

```diff
@@ -1,54 +1,61 @@
-import logging from '../../config/logging';
 import { User } from '../models/User';
+import { httpStatusCode } from '../../types/httpStatusCodes';
+import { logger } from '../../config/logger';
+import { APIError } from '../../errors/api-error';
 
-const NAMESPACE: string = 'repos/userRepos';
-
-// Get all accounts from table account
+//** Get all accounts from table account */
 const getAllUsers = async () => {
-    logging.info(`${NAMESPACE}: Querying all users.`);
-    await User.sync();
-    return User.findAll();
+    await User.sync({ logging: false });
+    return User.findAll({ logging: false });
 };
 
 // Create a sample user with hardcoded info (test-only)
 const createSampleUser = async () => {
-    logging.info(`${NAMESPACE}: Creating a sample user.`);
-    await User.sync({ alter: true });
+    await User.sync({ alter: true, logging: false });
     return await User.create({
         firstName: 'Sample',
         lastName: 'User',
         phone: '5145551237',
-        email: 'sample4@gmail.com'
+        email: 'sample4@gmail.com',
+        logging: false
     });
 };
 
-// Create a simple user with verified input
+//  Create a simple user with verified input
 const createSimpleUser = async (user: any) => {
-    logging.info(`${NAMESPACE}: Creating a simple user with request body.`);
-    await User.sync({ alter: true });
-    return await User.create({
+    await User.sync({ alter: true, logging: false });
+    const result = await User.create({
         firstName: user.firstName,
         lastName: user.lastName,
         email: user.email,
         countryCode: user.countryCode ? user.countryCode : undefined,
-        phone: user.phone ? user.phone : undefined
+        phone: user.phone ? user.phone : undefined,
+        logging: false
+    }).catch((error) => {
+        logger.error(error);
+        throw new APIError('This user already exists.', 'createSimpleUser', httpStatusCode.CONFLICT, true);
     });
```

Comment:
There could be so many Sequelize errors to be caught here though. The err message might not be correct.

Reply:
You're right, I will probably just add an if statement. And cast everything else in base error.

Reply:
I added a simple if statement, since the only validation error you can get with sequelize is a unique constraint error.
![image](https://user-images.githubusercontent.com/35941621/195356198-4c405a58-ed8f-464f-95c1-c2865606ddf7.png)